GroupIS1024/Models/Coach.cs

```diff
@@ -1,23 +1,29 @@
-﻿namespace GroupIS1024.Models
+﻿using Newtonsoft.Json;
+
+namespace GroupIS1024.Models
 {
     public class Coach
     {
+        [JsonProperty("Coach Id")]
         public int CoachId { get; set; }
 
+        [JsonProperty("First Name")]
         public string FirstName { get; set; }
 
+        [JsonProperty("Last Name")]
         public string LastName { get; set; }
 
-        public string Username { get; set; }
+        [JsonProperty("User Name")]
+        public string UserName { get; set; }
 
         public string State { get; set; }
 
-        public Boolean Memberstatus { get; set; }
-
-        public Team Team { get; set; }
+        [JsonProperty("Coach Status")]
+        public Boolean CoachStatus { get; set; }
 
+        public Team? Team { get; set; }
```

Comment:
Given the previous models. It is impossible to create either team, player, or coach first. All are interdependent on each other to be created. So, I have made foreign key properties nullable to resolve the chicken and egg problem

Reply:
One other thing I'd do here... It looks weird having a variable named Team of type Team.  Consider changing these somewhat; perhaps make the variable name lowercase to differentiate it.